src/Http/Senders/GuzzleSender.php

```diff
@@ -66,24 +66,37 @@ public function getFactoryCollection(): FactoryCollection
      */
     protected function createGuzzleClient(): GuzzleClient
     {
-        // We'll use HandlerStack::create as it will create a default
-        // handler stack with the default Guzzle middleware like
-        // http_errors, cookies etc.
-
-        $this->handlerStack = HandlerStack::create();
-
         // Now we'll return new Guzzle client with some default request
         // options configured. We'll also define the handler stack we
         // created above. Since it's a property, developers may
         // customise or add middleware to the handler stack.
```

Comment:
These comments should probably be updated a little bit. "Now" doesn't make sense anymore and the handler stack is not being created above anymore.